recipe/recipe_13TeV.sh

```diff
@@ -12,79 +13,48 @@ MINOR_VERSION=`echo $CMSSW_VERSION | sed "s|CMSSW_\([0-9]\)_\([0-9]\)_.*|\2|"`
 
 pushd $CMSSW_BASE/src
 
-echo "Checking out PAT dataformats"
-
-if [ "$LIMITS" = "1" ]
-then
-   echo ""     
-   echo "======================================="
-   echo "You shouldnt run limits/fits from here." 
-   echo "LIMITS should be run from 6XX, and do not require the rest of the machinery. Please, change area."
-   echo "Check https://twiki.cern.ch/twiki/bin/view/CMS/SWGuideHiggsAnalysisCombinedLimit for updates"
-   echo "======================================="
-   echo ""     
-fi
-
-
-if [ "$PATPROD" = "1" ]
-then
-  echo "Checking out tuple production tags"
-
-  echo "Need to update recipe for Quark Gluon Jet ID - which is the correct tag?"
-  #echo "Downloading Quark Gluon Jet ID"
-  #cvs co -r v1-2-3 -d QuarkGluonTagger/EightTeV UserCode/tomc/QuarkGluonTagger/EightTeV
-  # Quark-gluon tagging
-  git clone https://github.com/amarini/QuarkGluonTagger.git
-  pushd $CMSSW_BASE/src/QuarkGluonTagger
-  git checkout v1-2-6
-  pushd $CMSSW_BASE/src
+echo "Need to update recipe for Quark Gluon Jet ID - which is the correct tag?"
+#echo "Downloading Quark Gluon Jet ID"
+# Quark-gluon tagging
+git clone https://github.com/amarini/QuarkGluonTagger.git
```

Comment:
Maria requested a check for each of these merges and checkouts. That way you can rerun the recipe. I don't know how feasible this is in case something changes (such as the branch you merge). Perhaps we can just do a later PR. I don't have an obvious solution other than doing a directory check.